recipe/conda_build_config.yaml

```diff
@@ -659,6 +661,14 @@ s2n:
   - 1.3.28
 sdl2:
   - '2'
+sdl2_image:
+  - '2'
+sdl2_mixer:
+  - '2'
+sdl2_net:
+  - '2'
+sdl2_ttf:
+  - '2'
```

Comment:
hmmm. ok. i feel like we will forget to keep these in sync. but ok as well.

Reply:
They shouldn't _necessarily_ be kept in sync. It just "happens" that upstream uses the same major number and version policy for all of them. Also, it doesn't get much clearer than having all libs named with the same prefix and grouped in the CBC 😂